### PR TITLE
Add 0.197 release notes

### PR DESCRIPTION
**Pull request by electrum**

### CODE REVIEW

File: presto-docs/src/main/sphinx/release/release-0.197.rst [hunk 1]
```diff
@@ -0,0 +1,65 @@
+=============
+Release 0.197
+=============
+
+General Changes
+---------------
+
+* Fix query scheduling hang when the ``concurrent_lifespans_per_task`` session property is set.
+* Fix query failure when query contains a ``TIMESTAMP`` literal corresponding to a local time
+  that does not occur in the time zone of the Presto JVM. For example, if Presto was run in a
```
Comment (nezihyigitbasi) [inline presto-docs/src/main/sphinx/release/release-0.197.rst]: `was run` -> `was running`

File: presto-docs/src/main/sphinx/release/release-0.197.rst [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  CET zone (e.g., ``Europe/Brussels``) and the client session was in UTC, an expression
+  such as ``TIMESTAMP '2017-03-26 02:10:00'`` would cause a failure.
+* Extend predicate inference and pushdown for queries using a ``<symbol> IN <subquery>`` predicate.
+* Support predicate pushdown for the ``<column> IN <values list>`` predicate
+  where values in the ``values list`` require casting to match the type of ``column``.
+* Optimize :func:`min` and :func:`max` to avoid unnecessary object creation in order to reduce GC overhead.
+* Optimize the :func:`ST_XMin`, :func:`ST_XMax`, :func:`ST_YMin` and :func:`ST_YMax` functions.
```
Comment (nezihyigitbasi) [inline presto-docs/src/main/sphinx/release/release-0.197.rst]: comma before and

File: presto-docs/src/main/sphinx/release/release-0.197.rst [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+* Add :func:`ST_IsSimple` geospatial function.
+* Add support for broadcast spatial joins.
+
+Resource Groups Changes
+-----------------------
+
+* Change configuration check for weights in resource group policy to validate that
+  either all of the subgroups or none of the subgroups have a scheduling weight configured.
+* Add support for named variables in source and user regular expressions that can be
+  used to parameterize resource group IDs.
```
Comment (nezihyigitbasi) [inline presto-docs/src/main/sphinx/release/release-0.197.rst]: What is `resource group ID` (there is no ID property in the resource groups docs, so the users don't know it)?  I guess what we want to say is "resource group names".

File: presto-docs/src/main/sphinx/release/release-0.197.rst [hunk 4]
```diff
@@ -0,0 +1,65 @@
+=============
+Release 0.197
+=============
+
+General Changes
+---------------
+
+* Fix query scheduling hang when the ``concurrent_lifespans_per_task`` session property is set.
+* Fix query failure when query contains a ``TIMESTAMP`` literal corresponding to a local time
```
Comment (dain) [inline presto-docs/src/main/sphinx/release/release-0.197.rst]: `query` twice is a bit hard for me to read.  Maybe `Fix failure when a query contains a...`

File: presto-docs/src/main/sphinx/release/release-0.197.rst [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+  that does not occur in the time zone of the Presto JVM. For example, if Presto was running
```
Comment (dain) [inline presto-docs/src/main/sphinx/release/release-0.197.rst]: maybe `default time zone`?

File: presto-docs/src/main/sphinx/release/release-0.197.rst [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+  in a CET zone (e.g., ``Europe/Brussels``) and the client session was in UTC, an expression
+  such as ``TIMESTAMP '2017-03-26 02:10:00'`` would cause a failure.
+* Extend predicate inference and pushdown for queries using a ``<symbol> IN <subquery>`` predicate.
+* Support predicate pushdown for the ``<column> IN <values list>`` predicate
+  where values in the ``values list`` require casting to match the type of ``column``.
+* Optimize :func:`min` and :func:`max` to avoid unnecessary object creation in order to reduce GC overhead.
+* Optimize the :func:`ST_XMin`, :func:`ST_XMax`, :func:`ST_YMin`, and :func:`ST_YMax` functions.
```
Comment (dain) [inline presto-docs/src/main/sphinx/release/release-0.197.rst]: `function performance`?

File: presto-docs/src/main/sphinx/release/release-0.197.rst [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+* Add support for named variables in source and user regular expressions that can be
+  used to parameterize resource group names.
+
+Hive Changes
+------------
+
+* Fix reading of Hive partition statistics with unset fields. Previously, unset fields
+  were incorrectly interpreted as having a value of zero.
+* Fix integer overflow when writing a single file greater than 2GB with optimized ORC writer.
+* Fix system memory accounting to include stripe statistics size and
```
Comment (dain) [inline presto-docs/src/main/sphinx/release/release-0.197.rst]: maybe re-wrap line

File: presto-docs/src/main/sphinx/release/release-0.197.rst [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+  writer validation size for the optimized ORC writer.
+* Dynamically allocate the compression buffer for the optimized ORC writer
+  to avoid unnecessary memory allocation. Add config property
+  ``hive.orc.writer.max-compression-buffer-size`` to limit the maximum size of the buffer.
+* Add session property ``orc_optimized_writer_max_stripe_size`` to tune the
+  maximum stipe size for the optimized ORC writer.
+* Add session property ``orc_string_statistics_limit`` to drop the string
+  statistics when writing ORC files if they exceed the limit.
```
Comment (dain) [inline presto-docs/src/main/sphinx/release/release-0.197.rst]: `The default is 64 bytes`

**Pull request merged**